pkg/controllers/binding/binding_controller.go

```diff
@@ -92,7 +92,7 @@ func (c *ResourceBindingController) syncBinding(binding *workv1alpha1.ResourceBi
 		return controllerruntime.Result{Requeue: true}, err
 	}
 
-	err = helper.EnsureWork(c.Client, workload, clusterNames, c.OverrideManager, binding, apiextensionsv1.NamespaceScoped)
+	err = helper.EnsureWork(c.Client, workload, clusterNames, c.OverrideManager, binding, binding.Spec, apiextensionsv1.NamespaceScoped)
```

Comment:
It is a little strange that pass binding and binding.Spec as parameters at the same function.

Reply:
OK,I will modify it.